sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/model/SignResult.java

```diff
@@ -43,4 +50,44 @@ public String getValue() {
     public void setValue(String value) {
         this.value = value;
     }
+
+    @Override
+    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
+        jsonWriter.writeStartObject();
```

Comment:
Any reason `kid` and `value` aren't being serialized?

Reply:
Same as the Certificates class. I'll add it for good measure.